crates/router/src/core/payments/tokenization.rs

```diff
@@ -1,8 +1,6 @@
 use std::collections::HashMap;
 
 use ::payment_methods::controller::PaymentMethodsController;
-#[cfg(feature = "v1")]
-use api_models::payment_methods::PaymentMethodsData;
```

Comment:
Use `PaymentMethodsData` from domain Models. The inner variants of the enum still use API models, so it should not break. For Card, I have changed the domain model to be the same as the api model, as that is the version currently being stored.